tests/jvm/src/test/scala/scala/meta/tests/metacp/MetacpSuite.scala

```diff
@@ -1,10 +1,7 @@
 package scala.meta.tests.metacp
 
-import org.scalatest.Ignore
```

Comment:
Oops looks like this import wasn't unused 😅 

No idea how I didn't notice that.